From 2784bf5cfdecfa7a3bf751d93b8a05c70d13fbf1 Mon Sep 17 00:00:00 2001
From: ElXreno <elxreno@gmail.com>
Date: Fri, 19 Sep 2025 21:33:45 +0300
Subject: [PATCH] Fix building with -Werror=format-security

---
 src/App.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/App.cpp b/src/App.cpp
index a3d49b1..52eda7c 100644
--- a/src/App.cpp
+++ b/src/App.cpp
@@ -392,7 +392,7 @@ void App::_drawControls()
                     const auto draw_touch_sensor_combo = [&](auto& prop, const char* label) {
                         auto it = TOUCH_SENSOR_FUNCTION_STR.lower_bound({prop.desired});
                         if (it == TOUCH_SENSOR_FUNCTION_STR.end() || it->first != prop.desired)  {
-                            ImGui::Text(label);
+                            ImGui::TextUnformatted(label);
                             ImGui::SameLine();
                             ImGui::Text("Unknown Function: %d", static_cast<int>(prop.desired));
                             return;
-- 
2.51.0

